mod30/README.md

```diff
@@ -58,15 +77,16 @@ code.
 |[4 - Social Media Integration with Logic Apps ](demos.md#demo-4-social-media-integration-with-logic-apps)|[Link](https://globaleventcdn.blob.core.windows.net/assets/mod/mod30/MOD30_Demo4.mp4)
 |[5 - Automatic Image Captioning with Cognitive Services](demos.md#demo-4-automatic-image-captioning-with-logic-apps)|[Link](https://globaleventcdn.blob.core.windows.net/assets/mod/mod30/MOD30_Demo5.mp4)
 
-## Become a Presenter
+## Become a Trained Presenter
 
-To become a *Trained Presenter*, contact [scalablecontent@microsoft.com](mailto:scalablecontent@microsoft.com). In your email please include:
+To become a trained presenter, contact [learningpathmanager@microsoft.com](mailto:learningpathmanager@microsoft.com). In your email please include:
 
 - Complete name:
-- The code of this presentation: MOD30
-- Link (ex: unlisted YouTube video) to a video of you presenting (~10 minutes).
+- The code of this presentation: apps10
```

Comment:
The session code is MOD30, not apps10

Reply:
Will fix, missed when copied from apps10 example